src/Nri/Ui/Select/V4.elm

```diff
@@ -0,0 +1,97 @@
+module Nri.Ui.Select.V4 exposing
+    ( Config
+    , view
+    )
+
+{-| Build a select input.
+
+
+## Upgrading to V3
+
+  - Remove the old styles. V3 uses compile-at-runtime elm-css!
+
+
+# Configure
+
+@docs Config
+
+
+# Render
+
+@docs view
+
+-}
+
+import Css
+import Dict
+import Html.Styled as Html exposing (..)
+import Html.Styled.Attributes as Attributes exposing (..)
+import Html.Styled.Events exposing (..)
+import Json.Decode exposing (Decoder, andThen, succeed)
+import Nri.Ui
+import Nri.Ui.Colors.V1
+import Nri.Ui.Util
+
+
+{-| Select-specific Choice.Config
+-}
+type alias Config a =
+    { choices : List { label : String, value : a }
+    , current : a
+    , id : Maybe String
+    }
+
+
+{-| TODO: Consider moving this to Nri.Ui.Util once the non-0.19-approved `toString` is removed
+-}
+niceId : String -> a -> String
+niceId prefix x =
+    prefix ++ "-" ++ Nri.Ui.Util.dashify (Nri.Ui.Util.removePunctuation (toString x))
+
+
+{-| A normal select dropdown
```

Comment:
What does "normal" mean?  Can that qualification be dropped now that there are no other view functions in this module?